components/automate-ui/src/app/components/create-object-modal/create-object-modal.component.ts

```diff
@@ -43,12 +41,16 @@ export class CreateObjectModalComponent implements OnInit, OnChanges {
   }
 
   ngOnChanges(changes: SimpleChanges): void {
-    // if a new list of projects to populate dropdown with is passed in we update the dropdown
-    const checked = false;
-    if (hasIn('assignableProjects.currentValue', changes)) {
+    // update project dropdown if list changes
+    if (changes.assignableProjects) {
       this.projects = {};
       changes.assignableProjects.currentValue.forEach((proj: Project) =>
-        this.projects[proj.id] = { ...proj, checked });
+        this.projects[proj.id] = { ...proj, checked: false });
```

Comment:
PR extras: there are a variety of small cleanups such as the above in this PR unrelated to the issue being fixed.